pubsub/index.js

```diff
@@ -129,6 +149,10 @@ function setupCargo (endpoint, key) {
 function processMessage (cargo, message) {
   const { httpapi, identify } = parseMessage(message)
 
+  if (message.publishTime < Date.now() - WARNING_THRESHOLD) {
+    console.warn(timestamp(), 'Warning: Old message', { httpapi, identify })
```

Comment:
Ah yeah, only reason I didn't do this one was because I saw you left the other `console.warn` in there when you changed them in the other PR. Should that be a `console.log` too?

Reply:
oops, yes, I missed that one

Reply:
Done!